tests/framework/artifacts.py

```diff
@@ -305,7 +306,8 @@ def _fetch_artifacts(self,
                          artifact_ext,
                          artifact_type,
                          artifact_class,
-                         keyword=None):
+                         keyword=None,
```

Comment:
This is a generic method to fetch artifacts. I would not add specific behaviour here. You can do this in the firecrackers() method.

Reply:
good point

Reply:
done, PTAL